### PR TITLE
Added Desktop MP3 Player under Basic Python Desktop Application

### LINKED ISSUE #513: Add Desktop MP3 player Application in Python
### Is your feature request related to a problem? Please describe.

I think music players are a great source of help in playing various audio files. I'd like to suggest a desktop mp3 player to play audio files in your local machine. 

### Describe the solution you'd like.

An mp3 player using tkinter, pygame and mutagen modules to work as an aid to play digital audio files on your computer. 
I'd like to contribute this project to Desktop Application -> Basic -> Python, under GSSOC'22. 
Note: The issue and associated PR will not be duplicate. 

### Describe alternatives you've considered.

None. 

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes. 

### Description

- Added Desktop MP3 Player under Basic Python Desktop Application
- Made corresponding changes to README

Fixes #513 

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Screenshots

149654407-d9bdd55f-78cf-404f-b152-3085480b9c7e
149654409-9a109fb1-4845-4360-a320-8ef69852ee05
149654415-9cf532d7-b385-4893-95e3-5c7d2948e324
149654419-c3742cc4-2b2e-46c2-8f18-292040807352

### CODE REVIEW

File: Desktop Application/Basic/Python/Desktop MP3 Player/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Test Examples:
+
+![Screenshot (310)](https://user-images.githubusercontent.com/80174214/149654407-d9bdd55f-78cf-404f-b152-3085480b9c7e.png)
+
+![Screenshot (311)](https://user-images.githubusercontent.com/80174214/149654409-9a109fb1-4845-4360-a320-8ef69852ee05.png)
+
+![Screenshot (312)](https://user-images.githubusercontent.com/80174214/149654415-9cf532d7-b385-4893-95e3-5c7d2948e324.png)
+
+![Screenshot (313)](https://user-images.githubusercontent.com/80174214/149654419-c3742cc4-2b2e-46c2-8f18-292040807352.png)
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Desktop MP3 Player/README.md:39]: Please reduce the image size from this `readme.md` file .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Do the requires changes @abhilipsasahoo03

**Comment by abhilipsasahoo03** [comment]
Hi @Kushal997-das I've made the required changes kindly look into it.

**Review by Kushal997-das — APPROVED** [review]
@photon149 have a look !

**Comment by abhilipsasahoo03** [comment]
Hey @Kushal997-das @photon149 kindly review my PR and if there's any change needed to be made, let me know

**Comment by Kushal997-das** [comment]
> Hey @Kushal997-das @photon149 kindly review my PR and if there's any change needed to be made, let me know

@abhilipsasahoo03 It's under review only our mentor will review the PR and they will let you know .

**Review by photon149 — APPROVED** [review]
@abhilipsasahoo03 Everything seems perfect.
No Duplicity ✅ 
@Kushal997-das @prakhar3062 have a look

**Review by prakhar3062 — APPROVED** [review]

**Pull request merged**